packages/rs-platform-value/src/lib.rs

```diff
@@ -0,0 +1,963 @@
+//! Platform value
+//! A dynamic Platform value
+//!
+//! A module that is used to represent values in Platform components
+//! Forked from ciborium value
+//!
+//!
+pub mod btreemap_extensions;
+pub mod converter;
+pub mod display;
+mod error;
+pub mod inner_value;
+mod integer;
+pub mod system_bytes;
+pub mod value_map;
+
+pub use error::Error;
+pub use integer::Integer;
+
+pub type ValueMap = Vec<(Value, Value)>;
+pub type Hash256 = [u8; 32];
+
+/// A representation of a dynamic value that can handled dynamically
+#[non_exhaustive]
+#[derive(Clone, Debug, PartialEq, PartialOrd)]
+pub enum Value {
```

Comment:
maybe it's for the next step, but I would think of introducing the `Identifier` variant as this one has sometimes a different treatment, especially in terms of displaying (or) serializing.
actually, the concept of the package is similar to the projects I was working on someday ago:  https://github.com/qrayven/oxygen/blob/main/src/types/value.rs#L21

Reply:
Good idea